tests/Api/QuestionnaireReadTest.php

```diff
@@ -63,8 +63,21 @@ public function testAdminCanReadExistingQuestionnaireById(): void
         $this->authenticateAsAdmin();
 
         $questionnaire = Questionnaire::factory()->createOne();
+        $questionnaireModel = QuestionnaireModel::factory()->createOne();
+
+        Question::factory()
+            ->count(20)
+            ->create();
+
+        QuestionAnswer::factory()
+            ->count(20)
+            ->create([
+                'user_id' => $this->user->id,
+                'questionnaire_model_id' => $questionnaireModel->id
+            ]);
```

Comment:
Mybe you used something like this
```suggestion
        Question::factory()
            ->count(20)
            ->has(QuestionAnswer::factory()
            ->count(20)
            ->create([
                'user_id' => $this->user->id,
            ]))
            ->create();
```